versioned_docs/version-1.2/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Multiple Architecture Image Support
+sidebar_position: 110
+---
+
+# Multiple Architecture Image Support for ORAS
```

Comment:
Should the title be **Multi-platform Image/Artifact Support with ORAS**?

Reply:
I made this change, but I think "architecture" communicates more than "platform"